File: pytools/examples/golang/src/server/proto/req_chat_gm.go
package proto

import (
	"packet"
)

type ReqChatGm struct {
	content                  string
}

func ReqChatGmDecode(pack *packet.Packet) *ReqChatGm {
	reqChatGm := &ReqChatGm{}

	reqChatGm.content = pack.ReadString()
	return reqChatGm
}

func (this *ReqChatGm) GetContent() string {
	return this.content
}

File: pytools/data/data.code.csharp.server/ReqChatGm.cs
using System.Collections;
using System.Collections.Generic;


public class ReqChatGm
{
	private string _content;


	public ReqChatGm(Packet packet)
	{
		this._content = packet.ReadString();
	}


	public string content
	{
		get { return this._content; }
		set { this._content = value; }
	}

}
